scripts/bash/startup.sh

```diff
@@ -8,13 +8,13 @@ echo "Custom container startup"
 #
 ####################################################################################
 
-apt-get update -qq && apt-get install sendmail cron -yqq
+apt-get update -qq && apt-get install sendmail -yqq
 
 ####################################################################################
 #
 # Configure REDCap cronjob to run every minute
 #
 ####################################################################################
 
-service cron start
-(crontab -l 2>/dev/null; echo "* * * * * /usr/local/bin/php /home/site/wwwroot/cron.php > /dev/null")|crontab 
+# service cron start
```

Comment:
This probably should be re-enabled to ensure cron is configured even after scaling out or deploying a new container.